sdk/go/azure/azure.go

```diff
@@ -0,0 +1,19 @@
+// nolint: lll
+// Package azure offers support for all Microsoft Azure services and their properties.
+// Services are exposed as types from modules such as `appinsights`, `compute`, and `storage`, among many others. Using this
+// package allows you to programmatically declare instances of any Azure services using infrastructure as code, which
+// Pulumi then uses to drive the Azure API.
+//
+// If this is your first time using this package, these two resources may be helpful:
+//
+// * [Azure Getting Started Guide](https://pulumi.com/docs/quickstart/azure): Get up and running quickly.
```

Comment:
Super nit: this link has no `www` in it, the next line does

Reply:
Going to merge this and add this manually